lib/promise.flow

```diff
@@ -79,6 +79,11 @@ export {
 
 	//mapP and mapPparallel were moved to arrayutils.flow
 
+	// This function takes an array of promises and ties them together
+	// into one `bindP` expression. Meaning that all promises will be
+	// computed sequentially. Use it to avoid writing a `bindP` chain
+	// manually.
```

Comment:
document what happens when there is an error

Reply:
I noticed that there is `allP` function which does exactly the same as `sequenceP` but has a different implementation. It uses recursion and has optimization to keep a call stack from growing too much, while `sequenceP` uses `fold` from `array.flow`. I added `sequenceP` in order to save me some time of writing `bindP` chains. So I'm not sure which one is better and what to do about it. What do you think? @alstrup @dmitrybure

Reply:
if it's exactly the same, also in error handling, I would use allP